client/src/ui/explore/OpportunityTimeline.tsx

```diff
@@ -0,0 +1,89 @@
+"use client";
+import { Chrono } from "react-chrono";
+import "@/ui/explore/OpportunityTimeline.css";
+import { Typography } from "@mui/material";
+
+interface oppornityDataProps {
+  id: number;
+}
+export function OpportunityTimeline({ id }: oppornityDataProps) {
+  //future function for fetching timeline data
+  const fetchTimelineData = (oppId: number) => {
+    return console.log(oppId);
+  };
+
+  fetchTimelineData(id);
+
+  // Hard coded values for content card
+  const customContent = [
```

Comment:
Instead of storing the html in the array, map over the items array, that returns this element. This reduces redundancy